pyproject.toml

```diff
@@ -0,0 +1,6 @@
+# pyproject.toml
+[build-system]
+requires = ["setuptools<=65.5.0",
```

Comment:
The spacing looks a little funky here, is there a reason or should they fall on the same column?

Reply:
I copied this from Mirage without thinking too much about it. Maybe when I pasted the lines, the indentation was screwed up.

Reply:
Strange. The alignment in my local copy of the file is fine.

Reply:
Deleted some spaces, added some spaces. Now everything looks well-aligned.

Reply:
That is weird haha, thanks for fixing it!